go.mod

```
@@ -3,14 +3,14 @@ module github.com/google/go-sev-guest
 go 1.19
 
 require (
-	github.com/golang/glog v1.0.0
 	github.com/google/go-cmp v0.5.7
 	github.com/pborman/uuid v1.2.0
 	github.com/pkg/errors v0.9.1
 	go.uber.org/multierr v1.8.0
 	golang.org/x/crypto v0.0.0-20220525230936-793ad666bf5e
 	golang.org/x/sys v0.0.0-20220608164250-635b8c9b7f68
 	google.golang.org/protobuf v1.28.0
+	k8s.io/klog v1.0.0
```

```suggestion
	k8s.io/klog/v2 v2.80.1
```

Note that klog v2 is the latest release, which also fixes the GoLeak issue. klog v1 does not.

---

Done.